client/my-sites/email/titan-new-mailbox-list/style.scss

```
@@ -25,9 +25,21 @@
 		}
 	}
 
-	.titan-new-mailbox-list__actions {
+	.titan-new-mailbox-list__supplied-actions {
+		display: flex;
+		flex-direction: column;
+
 		@include break-mobile {
-			justify-content: space-between;
+			flex-direction: row;
+		}
+
+		> button:not( :first-child ) {
```

I just realised both this selector and the original on line 18 are using `button` -- I think this should be `.button` in both locations, as I believe some components may not render HTML `button` elements.
```suggestion
		> .button:not( :first-child ) {
```

---

Have addressed this.